src/controllers/EventsController.php

```diff
@@ -222,7 +223,12 @@ public function postAction($request, $db) {
                     $users = $user_mapper->getUserById($request->user_id);
                     $thisUser = $users['users'][0];
 
-                    $commentText = filter_var($request->getParameter('comment'), FILTER_SANITIZE_STRING);
```

Comment:
Quotes weren't displayed correctly.  I checked and talk comments don't use this filter. Either we remove the filter for comments or we make sure the filtered version is displayed correctly on screen.